documentation/visualization.md

```diff
@@ -115,5 +115,46 @@ To express that the dimension provides a specific _kind_ of data which is necess
 
 </aside>
 
+<section class="informative">
+
+##### Complex representation of time
+
+Representing temporal dimensions as literals typed as `xsd:date` or `xsd:dateTime` may be insufficient for complex scenarios. In such cases, data publishers may want to use the [Time Ontology in OWL](https://www.w3.org/TR/owl-time/) to represent temporal dimensions. This allows for the representation of complex temporal concepts, such as time periods, intervals, and repeating events, a well as joining cubes on the temporal dimension.
+
+When using Time Ontology in a cube, the values of a temporal dimension are resources of type `time:TemporalEntity`. The dimension should be expressed as a `time:GeneralDateTimeDescription` and have `Ordinal` scale type.
+
+<aside class='example'>
+
+```turtle
+<observation> a cube:Observation ;
+  time:hasTime <month/2000-06> .
+
+<month/2000-06> a time:ProperInterval ;
```

Comment:
@tpluscode probably a typo? 
`time:Interval` is used in the data i.e. at https://ld.admin.ch/time/month/1999-03

Reply:
No, it's actually intentional. [`time:ProperInterval`](https://prefix.zazuko.com/time:ProperInterval) is a specialisation of `time:Interval` such as that the start and end are two different. Non-zero duration.

I think we might actually make all Lindas intervals proper intervals

Reply:
this way it'll become consistent, sure 👍